guides/10_Video_Processor.md

```diff
@@ -0,0 +1,115 @@
+# Video Processing APIs
+
+## Introduction
+
+Amazon Chime SDK for JavaScript contains APIs to facilitate the processing of video stream and provides a simplified workflow for adding processing to the outgoing video feed.
+
+The APIs define a video processing stage as `VideoFrameProcessor`. A `VideoFrameProcessor` takes an array of `VideoFrameBuffer`s, applies the processing and outputs an array of `VideoFrameBuffer`s. Processors can be chained by their inputs and outputs. The outputs of each processor can be linked to the inputs of the next processor.
+Once the video is processed, it can be rendered onto a [HTMLCanvasElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement) and transformed into a [MediaStream](https://developer.mozilla.org/en-US/docs/Web/API/MediaStream).
+
+It is encouraged to use `VideoTransformDevice` to integrate `VideoFrameProcessor`s with meeting session. 
+`DeviceController.chooseVideoInputDevice` accepts `VideoTransformDevice` to transform the `Device` given the `VideoFrameProcessor`s.
+
+A simplified workflow would be:
+
+1. Create an array of `VideoFrameProcessor`s.
+2. Create `VideoTransformDevice` with input `Device` and the array of `VideoFrameProcessor`s.
+3. Call `meetingSession.audioVideo.chooseVideoInputDevice` with the `VideoTransformDevice`.
+
+
+### Browser compatibility
+
+The APIs for video processing in Amazon Chime SDK for JavaScript works in Firefox, Chrome, Chromium-based browsers (including Electron) on desktop and Android operating systems. A full compatibility table is below. Currently, it does not support Safari on iOS devices. WebRTC Unified Plan is also required. Unified Plan is by default enabled in [MeetingSessionConfiguration.enableUnifiedPlanForChromiumBasedBrowsers](https://aws.github.io/amazon-chime-sdk-js/classes/meetingsessionconfiguration.html#enableunifiedplanforchromiumbasedbrowsers).
```

Comment:
APIs is plural so 'it does' -> 'they do'

Reply:
What occurs if the APIs are called anyways?  Maybe `Safari on iOS does not support XXX, so attempting to call the APIs on those devices will XXX'